api/scripts/fill-in-assessment-method.js

```diff
@@ -0,0 +1,35 @@
+require('dotenv').config();
+
+const { knex } = require('../db/knex-database-connection');
+const logger = require('../lib/infrastructure/logger');
+
+async function fillInAssessmentMethod() {
+
+  const chunkSize = 50000;
+  const maxId = (await knex('assessments').max('id').first()).max;
```

Comment:
C'est intéressant : 
- les nouvelles insertions entre le début de script et la fin ne sont pas pris en compte, mais comme on les remplis correctement avant même de lancer le script il n'y pas de problème on traite bien tout. 
- On peut update des ids au dessus du maxId initialement choisie (due au chunk) 

Mais ça me semble pas inquiétant.

Reply:
> Il faut donc quand même attendre la fin de ce script pour commencer à faire des campagnes FLASH au risque d'avoir la méthode repasser à SMART_RANDOM
@VincentHardouin On peut ajouter une condition du genre `where "method" is null` pour ne mettre à jour que les lignes qui n'ont pas encore de `method`